src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -194,13 +194,23 @@ private void processRequests(HttpServerRequest httpServerRequest) {
                     this.httpSinkEndpoints.remove(deleteInstanceID);
                 } else {
                     httpServerRequest.response()
-                            .setStatusCode(404)
+                            .setStatusCode(ErrorCodeEnum.NOT_FOUND.getValue())
+                            .setStatusMessage("endpoint not found")
                             .end();
                 }
                 break;
-
+            case EMPTY:
+                httpServerRequest.response()
+                        .setStatusCode(ErrorCodeEnum.UNPROCESSABLE_ENTITY.getValue())
+                        .setStatusMessage("records cannot not be empty")
```

Comment:
I still see "records" in this message. But it could be not a POST request about records but about offsets for example.